docs/core/install/linux-rhel.md

```diff
@@ -44,13 +41,19 @@ The following table is a list of currently supported .NET releases on both RHEL
 
 [!INCLUDE [package-manager uninstall notice](./includes/linux-uninstall-preview-info.md)]
 
-## RHEL 8 ✔️
+## RHEL 9
+
+.NET is included in the AppStream repositories for RHEL 9.
+
+[!INCLUDE [linux-dnf-install-70](includes/linux-install-70-dnf.md)]
+
+## RHEL 8
 
 .NET is included in the AppStream repositories for RHEL 8.
 
-[!INCLUDE [linux-dnf-install-60](includes/linux-install-60-dnf.md)]
+[!INCLUDE [linux-dnf-install-70](includes/linux-install-70-dnf.md)]
 
-## RHEL 7 ✔️ .NET 6.0
+## RHEL 7 ✔️ .NET 6
```

Comment:
I see the checkmark is removed in other places. Is this just left over?

Reply:
I left here just because it acts as a good visual separator between the acronym and .NET